{
  "cveId": "CVE-2025-8291",
  "eventName": "CVE Modified",
  "cveChangeId": "C0EDA5E0-3F70-434D-9642-897924711B7F",
  "sourceIdentifier": "cna@python.org",
  "created": "2025-10-07T19:15:39.503",
  "details": [
    {
      "action": "Added",
      "type": "Reference",
      "newValue": "https://github.com/python/cpython/commit/162997bb70e067668c039700141770687bc8f267"
    },
    {
      "action": "Added",
      "type": "Reference",
      "newValue": "https://github.com/python/cpython/commit/333d4a6f4967d3ace91492a39ededbcf3faa76a6"
    },
    {
      "action": "Added",
      "type": "Reference",
      "newValue": "https://github.com/python/cpython/issues/139700"
    }
  ]
}